apps/apps.h

```diff
@@ -632,4 +635,12 @@ typedef struct verify_options_st {
 
 extern VERIFY_CB_ARGS verify_args;
 
+int istext(int format);
+const char *modestr(char mode, int format);
+
+#ifdef _WIN32
+int WIN32_rename(const char *from, const char *to);
+# define rename(from,to) WIN32_rename((from),(to))
+#endif
```

Comment:
You must remove `static` from the definition of `WIN32_rename` in apps.c as well.

Reply:
Damn, I thought this function is a standard WIN32 API...